### ISSUE TITLE
2 hubs, both as TV accessories - can only publish accessory as switch on first hub

### ISSUE THREAD

**Issue by mbial** [nicoduj/homebridge-harmony#189]
I am running 2 hubs.  Both are using default "TVAccessory": true.  

If I use "activitiesToPublishAsAccessoriesSwitch" for a specific activity, it works for ONLY the first listed hub in my config file.

If I add "activitiesToPublishAsAccessoriesSwitch" for a specific activity on the second-listed hub, it doesn't publish.
<img width="610" alt="Screen Shot 2019-09-17 at 1 51 44 PM" src="https://user-images.githubusercontent.com/8181777/65067102-bbee9980-d953-11e9-8d26-c85ed29df79a.png">

**Comment by nicoduj** [comment]
hi,

syntax seems to be bad for activitiesToPublishAsAccessoriesSwitch, it needs to be an **array**, like that : 

"activitiesToPublishAsAccessoriesSwitch": ["Listen To Sonos"]

**Comment by mbial** [comment]
Thank you for the quick response. Unfortunately, when I make the suggested change (and this time included the "activitiesToPublishAsAccessoriesSwitch": ["Listen To Sonos"] for the second hub) I have the same result; only the first hub publishes the specified activity.
<img width="543" alt="Screen Shot 2019-09-17 at 2 33 30 PM" src="https://user-images.githubusercontent.com/8181777/65069187-273a6a80-d958-11e9-9c74-7dd123163349.png">

**Comment by nicoduj** [comment]
Can you share your logs (in debug mode if possible) ? Are you sure of the name of the activity on the second hub ? Is it published well on the second TV accessory ?

**Comment by mbial** [comment]
Found my error.  Definitely not a bug on your end.  The name of the activity on the second hub was "Listen to Sonos" with a lower case "t".  So the activities are case sensitive!!  Once I changed the name on my Harmony hub to match "Listen To Sonos" with a capital "T" and restarted Homebridge, everything is working perfectly.  Thanks for all your work on this - it's a great plugin that adds a lot of function to HomeKit.
